spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -17,6 +17,7 @@
            study_mode: :full_time,
            subject_count: 0,
            subjects: [course_subject],
+           program_type: 'TA',
```

Comment:
Can we add the known fields as part of the course factory as a trait, along with a default one?
It be nice capture `random chars` to actual meanings

Reply:
Done, good suggestion